site/topics/numbers/number-bases.rst

```diff
@@ -35,7 +35,7 @@ Counting in Base Ten
 
 * Counting in base ten consists of moving to the next symbol
 
-    .. list-table:: Counting With Nine Symbols
+    .. list-table:: Counting with Nine Symbols
```

Comment:
Hang on. It's ten symbols.

Reply:
Wait.... how does this evenhappen?

Reply:
At first, I assumed you were making a bad `10` joke (2 vs 10).